src/components/team/Team.js

```diff
@@ -0,0 +1,56 @@
+import logo from '../../images/indeed.svg';
+import { SectionTitle } from '../sharedComponents/sharedComponents';
+import { Link, LinkImg, Partner, PartnerLink, PartnerLInk, PartnerRow1, PartnerRow2, TeamBlock, TeamBlocks, TeamImage, TeamText, TeamWrapper } from './styledTeam';
+import { TeamData } from '../../constants/teamConst';
+
+const {
+    teamTitle,
+    members,
+    partnersTitle,
+    partners1,
+    partners2
+} = TeamData;
+
+const Team = () => {
+
+    return (
+        <TeamWrapper>
+            <SectionTitle>{teamTitle}</SectionTitle>
+            <TeamBlocks>
+                {members.map((member) => {
+                    return (
+                        <TeamBlock key={members.indexOf(member)}>
+                            <TeamImage src={member.image} />
+                            <TeamText>{member.name}</TeamText>
+                            <TeamText>{member.role}</TeamText>
+                            <Link
+                                href={member.link}
+                                target='_blank' />
+                        </TeamBlock>
+                    )
+                })}
+            </TeamBlocks>
+            <SectionTitle>{partnersTitle}</SectionTitle>
+            <PartnerRow1>
+                {partners1.map((part) => {
+                    return (
+                        <PartnerLink href={part.link} target='_blank'>
```

Comment:
One thing I notice is you can click to follow the link far away from the image on some of the partners. One solution could be to use javascript with window.open(). https://stackoverflow.com/questions/4907843/open-a-url-in-a-new-tab-and-not-a-new-window